src/style.css

```diff
@@ -1 +1,46 @@
-/* add styles here */
+body,
+html {
```

Comment:
Avoid using tag names for styling elements directly. It's better to use classes to style elements to ensure that the styles are more modular and don't have unintended side effects. For example, you could use a class like `.page` for the `html` and `body` tags.